### PR TITLE
etcd-operator-cluster stability fixes

### PR DESCRIPTION
**Pull request by drpebcak**
fixes for telemetry chart

add containerPort for git-mirror

### CODE REVIEW

File: charts/telemetry/0.1.0/values.yaml [hunk 1]
```diff
@@ -35,11 +35,11 @@ ingress:
...
 resources:
   limits:
-   cpu: 300m
-   memory: 512Mi
+   cpu: 100m
+   memory: 128Mi
   requests:
-   cpu: 300m
-   memory: 512Mi
+   cpu: 100m
```
Comment (cloudnautique) [inline charts/telemetry/0.1.0/values.yaml]: where are these coming from ?

Comment (drpebcak) [reply charts/telemetry/0.1.0/values.yaml]: They are mostly just made up, but they are based on the fact that cpu/mem usage in production seems to be pretty low.

Comment (cloudnautique) [reply charts/telemetry/0.1.0/values.yaml]: lets double the limit.

Comment (drpebcak) [reply charts/telemetry/0.1.0/values.yaml]: Done, see latest commit.

**Pull request merged**